pkgs/applications/video/mpv/default.nix

```
@@ -105,6 +105,11 @@ in stdenv.mkDerivation rec {
     sha256 = "138921kx8g6qprim558xin09xximjhsj9ss8b71ifg2m6kclym8m";
   };
 
+  patches = optional archiveSupport (fetchpatch {
+    url = "https://patch-diff.githubusercontent.com/raw/mpv-player/mpv/pull/6438.patch";
```

```suggestion
    url = "https://github.com/mpv-player/mpv/commit/4b2f25822a70d82afe48aeb9e2c68ccd7aa8c86c.patch";
```

Pretty sure PR diffs aren't stable until they're merged.

---

Fair enough; I was basing this off of other `patch-diff` usages I grepped before submitting this and thought it might have become preferred.